src/model/Riddle.ts

```diff
@@ -0,0 +1,13 @@
+
+class Riddle {
+  regex: Regex
+  answers: Array<string>
+
+  // whether this riddle is solved correctly if the regex matched
+  solvedOnMatch: boolean = true
```

Comment:
I'm more a fan of using `enum`s in this case than of using `boolean`s.

Reply:
Also, when is this variable set at all?

Reply:
switched to enum,
this variable will be used when inverted questions are implemented.
(eg asking which regex does **not** match)